src/data/nat/gcd/basic.lean

```diff
@@ -6,7 +6,10 @@ Authors: Jeremy Avigad, Leonardo de Moura
 import algebra.group_power.order
 
 /-!
-# Definitions and properties of `gcd`, `lcm`, and `coprime`
+# Definitions and properties of `nat.gcd`, `nat.lcm`, and `nat.coprime`
+
+The more general equivalents of these are provided in a later file as `gcd_monoid.gcd`,
+`gcd_monoid.lcm`, and `is_coprime`.
```

Comment:
@alreadydone, github won't let me apply your suggestion above. Are you happy with: 

```suggestion
The more general equivalents of these are provided in a later file as `gcd_monoid.gcd` and
`gcd_monoid.lcm`.

Note that the global `is_coprime` is not a straightforward generalization of `nat.coprime`, see
`nat.is_coprime_iff_coprime` for the connection between the two.
```
or shall I drop the last statement.

Reply:
Looks good except I think the phrase "general equivalents" was invented to refer to `nat.is_coprime_iff_coprime` being iff but not defeq, but since `is_coprime` is no longer in the first sentence, you could just say "generalizations".

I [mentioned the coercion to Z](https://github.com/leanprover-community/mathlib/pull/17131#discussion_r1008954570) but maybe I was trying to convey too much in that message ... :)